Dockerfile

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.8
+FROM alpine:3.15.6
```

Comment:
Is there a special reason for not using alpine `3.16.2`?

Reply:
Yes, for 3.16 alpine npm package is not released yet.

Reply:
It is not available in main package of 3.16, but in the community package `https://dl-cdn.alpinelinux.org/alpine/v3.16/community/aarch64/npm-8.10.0-r0.apk`. I thought npm might have been the motivation behind adding it.

For 3.15.6 npm is available in the main package. In this case we do not need to add the community package at all.

I'm happy with both options.

Reply:
For now, I will go with current changes as In the future anyway, we want to switch to the upstream image.

Reply:
maybe there was a misunderstanding 😄 
I created a follow-up comment for the 3.15.6 case.